### DISCUSSION TITLE
Default collapsed/expanded state for collapsible field

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/payload#1376] [Feature Requests & Ideas]
I have 1 collapsible field that I would like to collapse by default

**Comment by denolfe** [comment]
This functionality exists. You can set `initCollapsed` to true under the field's admin options. Documentation here: https://payloadcms.com/docs/fields/collapsible

**Reply by wrobson-lllow** [reply]
initCollapsed doesn't seem to work the way I would expect it.  I included a collapsible field in a recent deployment and it is defaulting to open when a user goes to that page for the first time, once it is manually collapsed, payload seems to remember the next time you visit the page but not always.

**Reply by revnelson** [reply]
This is happening for me, as well.
